docs/lofi-hifi-backends.md

```diff
@@ -0,0 +1,207 @@
+# Introduce Instructlab profiles managed via `ilab profile...` to run different key commands at different Fidelity levels
```

Comment:
I think it's worth changing the filename to reflect the latest approach.

Reply:
This first line reads more like a summary than a title. I would shorten it to something like:
```suggestion
# InstructLab Training Profiles
```